src/style.css

```
@@ -1,3 +1,64 @@
-body {
+html {
+  font-family: Roboto, sans-serif;
+  --blue_color: #00acdc;
+  font-size: 12px;
+}
+
+body,
+ul {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.moyo_logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  text-align: center;
+}
+
+.nav__item {
+  display: flex;
+  margin-left: 20px;
+  text-transform: uppercase;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
```

1) you can simplify adding margin
2) move text-transform: uppercase; to a link
```suggestion
.nav__item:not(:first-child) {
  margin-left: 20px;
}
```